web/src/engine/websites/HentaiRead.ts

```diff
@@ -3,26 +3,31 @@ import icon from './HentaiRead.webp';
 import { DecoratableMangaScraper } from '../providers/MangaPlugin';
 import * as Common from './decorators/Common';
 
-const pageScript = `new Promise(resolve => {
-    const pagelist = chapter_preloaded_images;
-    resolve(pagelist.map(link => {
-        let uri = new URL(link);
-        uri.searchParams.set('quality', '100');
-        uri.searchParams.delete('w');
-        return uri.href;
-    }));
-});
+const pageScript = `
+    new Promise((resolve, reject) => {
+        try {
```

Comment:
No need for `try ... catch` in non-async sequential callback.
Errors are captured directly by the promise and will reject it.

Make sure to understand the capture scope for errors within a promise and use `try ... catch` only when appropriate or necessary
```javascript
// Bad (redundant)
new Promise((resolve, reject) => {
    // Errors within promise scope can be manually captured => manually rejected
    try {
        throw new Error();
        resolve();
    } catch(error) {
        reject(error);
    }
});

// Good
new Promise(resolve => {
    // Errors within promise scope are captured automatically => automatically rejected
    throw new Error();
    resolve();
});

// Bad
new Promise(resolve => {
    // Errors outside of promise scope are not captured automatically => never rejected
    setTimeout(() => {
        throw Error();
        resolve();
    }, 0);
});

// Good
new Promise((resolve, reject) => {
    // Errors outside of promise scope must be captured manually => manually rejected
    setTimeout(() => {
        try {
            throw Error();
            resolve();
        } catch(error) {
            reject(error);
        }
    }, 0);
});

// Bad
new Promise(async (resolve) => {
    // Errors outside of promise scope are not captured automatically => never rejected
    throw Error();
    resolve();
});

// Good
new Promise(async (resolve, reject) => {
    // Errors outside of promise scope must be captured manually => manually rejected
    try {
        throw Error();
        resolve();
    } catch(error) {
        reject(error);
    }
});
```

Reply:
I see, its a scope problem. I think i get it now. Sorry about that.